### DISCUSSION TITLE
사용자 관련 요청 테이블 추가 필요

### DISCUSSION THREAD

**Discussion by JuneParkCode** [Kernel360/hackathon2-gyeonggiBus#11] [General]
@km2535 @limbaba1120 @ss0ngcode @JuneParkCode 

## 사용자 요청 테이블
- 현재 사용자가 데이터 수집 요청을 하게 됩니다. 이 때, 사용자의 데이터 수집 요청을 저장하는 테이블이 필요합니다.
- 필요한 칼럼은 다음과 같습니다.
  - PK
  - 정류장 ID
  - 노선 ID
  - 데이터 수집을 원하는 시간

- 사용자에게서 데이터 수집을 원하는 시간대를 요청받게 된다고 가정한다면, api request 수를 줄일 수 있습니다.
 
- 예를 들어,  사용자는 17:00 에 사당역 10번 출구의 7800 번 데이터 수집을 원합니다.
 - 데이터 수집 요청을 받은 서버는 해당 시간에서 전후 30 분의 결과를 주기적으로 수집합니다.
   - 위 예시에서는 16:30 ~ 18:30 의 시간동안만 해당 요청에 대해서 데이터 수집을 수행합니다.
 - 결과적으로 배차 간격이 10분이었다고 한다면, 약 16:45 ~ 17:15 정도의 정류장 좌석 정보를 획득할 수 있습니다. 

ERD 에 해당 테이블 반영하고, API 명세에 작성해두겠습니다.
즐거운 주말 되세요~

**Comment by limbaba1120** [comment]
주말동안 OpenAPI 에서 받은 데이터와 csv 파일을 mysql에 저장해보겠습니다.
사용자 요청 테이블은 이해는 되는데 월요일에 다시 한번 얘기해보고 싶습니다!

**Reply by JuneParkCode** [reply]
네 확인했습니다. 주말 간에 조사를 하면서 발견한 점들이 있으면 바로 이야기해보도록합시다.
